src/Countries/Albania.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Albania extends Country
+{
+    public function countryCode(): string
+    {
+        return 'al';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Viti i Ri' => '01-01',
+            'Dita e Verës' => '03-14',
+            'Dita e Sulltan Nevruzit' => '03-22',
+            'Dita Ndërkombëtare e Punëtorëve' => '05-01',
+            'Dita e Shenjtërimit të Shenjt Terezës' => '09-05',
+            'Dita e Pavarësisë' => '11-28',
+            'Dita e Çlirimit' => '11-29',
+            'Dita Kombëtare e Rinisë' => '12-08',
+            'Krishtlindja' => '12-25',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable|string> */
+    protected function variableHolidays(int $year): array
+    {
+        return [
+            'E diela e Pashkëve Katolike' => $this->easter($year),
+            'E diela e Pashkëve Ortodokse' => $this->orthodoxEaster($year),
+            'Dita e Bajramit të Madh' => $this->getEidAlFitrHoliday($year),
+            'Dita e Kurban Bajramit' => $this->getEidAlAdhaHoliday($year),
+        ];
+    }
+
+    /**
+     * 
+     */
+    private function getEidAlFitrHoliday(int $year): string
```

Comment:
```suggestion
    private function getEidAlFitrHoliday(int $year): ?string
```

Reply:
@fitimvata that could work as well, changes pushed.